.github/workflows/markdownlint/config.yaml

```
@@ -0,0 +1,11 @@
+# All rules are inactive by default.
+default: false
+
+# These specific rules are active.
+# See https://github.com/DavidAnson/markdownlint#rules--aliases for details.
+heading-increment: true
+no-reversed-links: true
+no-missing-space-atx: true
+no-multiple-space-atx: true
+blanks-around-lists: true
+no-alt-text: true
```

```suggestion
no-alt-text: true
titlecase-rule: true
```
We need to explicitly enable the titlecase-rule, otherwise it won't take affect (due to `default: false`)